src/tools/support_documentation_tools.py

```
@@ -0,0 +1,2298 @@
+            )
+
+    except Exception as e:
+        processing_time = (datetime.now() - start_time).total_seconds()
+        logger.error(f"Error in ingest_markdown_docs: {str(e)}", exc_info=True)
+
+        return json.dumps(
+            {"success": False, "error": f"Unexpected error: {str(e)}", "processing_time": processing_time}, indent=2
+        )
+
+
+async def search_support_docs(
```

Pylint (3.3.7)

[refactor] Too many arguments (16/5)

(R0913)